{"name": "Ошибка, при создании курьера с пустым обязательным полем", "status": "broken", "statusDetails": {"message": "TypeError: 'dict' object is not callable", "trace": "self = <tests.test_create_courier.TestCreateCourier object at 0x000001EC294C1A60>, create_new_courier = {'firstName': 'tnlpuudvnu', 'login': 'ueaitydmpg', 'password': 'dsrjavfqji'}\n\n    @allure.title(\"Ошибка, при создании курьера с пустым обязательным полем\")\n    def test_create_courier_with_null_login_error_400(self, create_new_courier):\n>       payload = create_new_courier(10)\nE       TypeError: 'dict' object is not callable\n\ntests\\test_create_courier.py:22: TypeError"}, "start": 1717365865386, "stop": 1717365865394, "uuid": "195fa412-dcd7-4f24-9bd6-897d4cf02a11", "historyId": "6d12d9da8e6ce0466cb9eca08852569c", "testCaseId": "6d12d9da8e6ce0466cb9eca08852569c", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_with_null_login_error_400", "labels": [{"name": "story", "value": "Создание курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "Rayless"}, {"name": "thread", "value": "117356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}